config/config.go

```diff
@@ -102,6 +99,8 @@ type Hedera struct {
 	BridgeAccount string   `yaml:"bridge_account" env:"VALIDATOR_CLIENTS_HEDERA_BRIDGE_ACCOUNT"`
 	PayerAccount  string   `yaml:"payer_account" env:"VALIDATOR_CLIENTS_HEDERA_PAYER_ACCOUNT"`
 	TopicId       string   `yaml:"topic_id" env:"VALIDATOR_CLIENTS_HEDERA_TOPIC_ID"`
+	FeePercentage int64    `yaml:"fee_percentage" env:"VALIDATOR_CLIENTS_HEDERA_FEE_PERCENTAGE"`
```

Comment:
I think that if its `int64` we should be using the same concept for precision as in the contracts. If we are not going to use that concept, we need to be using float most probably in order to represent f.e fee of `1.9%`